app/services/intercom_service.rb

```diff
@@ -0,0 +1,113 @@
+require 'intercom'
+
+class IntercomService
+  def self.create_intercom_message_from_email(incoming_email, fw_from_hub: false)
```

Comment:
It seems like there is a lot of "consistent state" for this object (like the email and client, etc) that get used amongst all of these class methods. To me, it seems like creating an instance of the intercom service in the controller and calling the send method on an instance would be prudent because it would set this class up in a way that could share instance variables without passing around parameters to methods. However, calling a class method off of a service is also an established pattern, so this also makes sense. Just something to think about in terms of class design I guess.